hazel/third_party/cabal2bazel/bzl/cabal_package.bzl

```diff
@@ -371,8 +370,13 @@ def _get_build_attrs(
             # Collect direct cbits dependencies.
             cdeps[condition] += [hazel_cbits(p)]
             if p in _CORE_DEPENDENCY_INCLUDES:
-                cdeps[condition] += [_CORE_DEPENDENCY_INCLUDES[p]]
-                deps[condition] += [_CORE_DEPENDENCY_INCLUDES[p]]
+                # Create per-package unique import of core dependency.
+                haskell_toolchain_library(
+                    name = _CORE_DEPENDENCY_INCLUDES[p],
```

Comment:
Hazel generates `haskell_toolchain_library` and `cc_library` targets for all items of `core_packages`. This includes `unix`, and `rts`. So, they should be available as `@haskell_unix//:unix(-cbits)` and `@haskell_rts//:rts(-cbits)`. However, the `-cbits` targets are empty dummy targets. Now that `haskell_toolchain_library` contains a `CcInfo`, we can probably drop much of the `cbits` logic.